starter/src/app/_components/universal/layout.tsx

```diff
@@ -0,0 +1,47 @@
+import { cn } from "@/lib/utils";
+import React from "react";
+
+type BaseProps = {
+  children: React.ReactNode;
+  className?: string;
+};
+
+type LayoutProps = BaseProps & {
+  flex?: boolean;
+  fullWidth?: boolean;
+  align?: "center" | "start" | "end";
+};
+
+export const LayoutAside = ({ children, className }: BaseProps) => {
+  return <aside className={`layout-aside min-w-[260px] p-4 ${className}`}>{children}</aside>;
+};
+
+export const LayoutMain = ({ children, className }: BaseProps) => {
+  return <main className={`layout-main p-4  ${className}`}>{children}</main>;
```

Comment:
Using `cn` here avoids any css specificity issues from manual className interpolation.


```suggestion
  return <main className={cn("layout-main p-4", className)}>{children}</main>;
```